openedx/features/course_experience/static/course_experience/js/CoursetalkReviews.js

```diff
@@ -0,0 +1,28 @@
+/*
+  Enable users to switch between viewing and writing CourseTalk reviews.
+ */
+
+export class CourseTalkReviews {  // eslint-disable-line import/prefer-default-export
```

Comment:
I know that all of our other Webpack files do this, but I'm wondering why. It seems concerning that we're not following best practices.

https://github.com/benmosher/eslint-plugin-import/blob/master/docs/rules/prefer-default-export.md